frontend/src/navigation_header.tsx

```diff
@@ -0,0 +1,94 @@
+
+import * as React from 'react'
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import HomeIcon from '@material-ui/icons/Home';
+import AccountBalanceIcon from '@material-ui/icons/AccountBalance';
+import VpnKeyIcon from '@material-ui/icons/VpnKey';
+import MeetingRoom from '@material-ui/icons/MeetingRoom';
+import GroupIcon from '@material-ui/icons/Group';
+import {Login} from './login'
+import {Genebanks} from './genebanks'
+import {Manage} from './manage'
+import {Switch, Route} from 'react-router-dom'
+import {useUserContext} from './user_context'
+import * as ui from './ui_utils'
+
+// Define styles for tab menu
+const useStyles = makeStyles({
+  root: {
+    flexGrow: 1,
+  },
+  main: {
+    height: "calc(100% - 72px)",
+  }
+});
+
+export function NavigationHeader() {
```

Comment:
There is no need for this DataContext-free Navigation anymore, right?

Reply:
It must have been added again after some merge. I removed it again